docs/src/theme/NotFound/Content/index.js

```diff
@@ -1,32 +1,26 @@
-import React from 'react';
-import clsx from 'clsx';
 import Translate from '@docusaurus/Translate';
 import Heading from '@theme/Heading';
-export default function NotFoundContent({className}) {
+import clsx from 'clsx';
+import React from 'react';
+export default function NotFoundContent({ className }) {
   return (
     <main className={clsx('container margin-vert--xl', className)}>
       <div className="row">
         <div className="col col--6 col--offset-3">
           <Heading as="h1" className="hero__title">
-            <Translate
-              id="theme.NotFound.title"
-              description="The title of the 404 page">
+            <Translate id="theme.NotFound.title" description="The title of the 404 page">
               Oh no! There's nothing here.
             </Translate>
           </Heading>
           <p>
-            <Translate
-              id="theme.NotFound.p1"
-              description="The first paragraph of the 404 page">
-              If we've led you astray, please let us know. We'll do our best to get things in order. 
-
+            <Translate id="theme.NotFound.p1" description="The first paragraph of the 404 page">
+              If we've led you astray, please let us know. We'll do our best to get things in order.
```

Comment:
this might be more of a copy nit for a follow up PR, but should we include some sort of link or CTA here so folks can leave feedback on the docs?

Reply:
Great call. @hello-ashleyintech perhaps you can help me, a total React noob, here: I tried wrapping the "please let us know" text in an `<a href=""></a>` tag and link to this repo's issue creation, but I get a, I think, React error 😢 

How.. does on do links in React? 👶 

<img width="1217" alt="Screenshot 2024-10-04 at 9 55 36 AM" src="https://github.com/user-attachments/assets/2a05c429-b0a2-4942-b5f8-29e3b179f04e">

Reply:
ooooh yes! That looks like it might actually be caused by embedding a link in the `<Translate>` tag from Docusaurus. It seems like that tag only supports simple strings. The suggested workaround in the docs is to break this into substrings wrapped in the `<Translate>` tag - maybe to keep that link as part of the translation it could be something like:

```
<Translate>If we've led you astray, </Translate><a href=""><Translate>please let us know</Translate></a><Translate>. We'll do our best to get things in order.</Translate>
```

If that doesn't work though then we might need to just not translate the `please let us know` line for now 😬